morf-core/src/test/java/org/alfasoftware/morf/jdbc/TestSqlScriptExecutor.java

```diff
@@ -52,6 +52,65 @@ public void setUp() throws SQLException {
     when(databaseType.reclassifyException(any(Exception.class))).thenAnswer(invoc -> (Exception) invoc.getArguments()[0]);
   }
 
+  /**
+   * Verify that each {@link SqlScriptExecutor} sets the correct fetch sizes {@link java.sql.PreparedStatement}.
+   */
+  @Test
+  public void testDefaultFetchSizesForSqlScriptExecutor() throws SQLException {
+    verify(sqlDialect, times(1)).fetchSizeForBulkSelects();

```

Comment:
Shouldn't we instead verify the fetch size being set on the prepared statement?

Reply:
Discussed with Martyn - leaving this as is as there's not a clean way to assert on this class's private field (fetchSizeForBulkSelects) or the value set on the inner NamedParameterStatement without using something like Powermock.